ssn/ssn_separated/ssn.ttl

```diff
@@ -670,9 +658,9 @@ ssn:Observation rdf:type owl:Class ;
                                   owl:minCardinality "0"^^xsd:nonNegativeInteger
                                 ] ,
                                 [ rdf:type owl:Restriction ;
-                                  owl:onProperty ssn:featureOfInterest ;
+                                  owl:onProperty sosa:featureOfInterest ;
```

Comment:
These changes look all good to me. We can discuss the "missing" link between FeatureOfInterest and xxxProperty in our next phone call. I put quotation marks here, because I think it was deliberately left out from SOSA.
In another note, what is the oldssn.ttl in your directory though? Our intention was to only update ssnx.ttl, the alignment between old ssn and new ssn. We don't want to update old SSN at all.